### ISSUE TITLE
The score on training set is very different from the one on validation set

### ISSUE THREAD

**Issue by ewrfcas** [knowledgetechnologyuhh/OMGEmotionChallenge#4]
I can get 0.78 score in training set while the one is only 0.04 on validation set. What's wrong with my method? Who can give me some advices?

**Comment by EgorLakomkin** [comment]
Hi @ewrfcas and thank you for your interest in this challenge! Maybe the gap you observe can be due to model overfitting and some regularization methods can help (various variants of dropout, batch/layer/weight normalization). Do you monitor the model performance during training on the held out dataset?

**Comment by ewrfcas** [comment]
@EgorLakomkin Thank you for your advices. Unfortunately, the loss of the held out dataset achieves no improvements after the first epoch.

**Comment by pablovin** [comment]
Dear @ewrfcas , we had some problems with the cccalculation script, could you update the script and try it again now? maybe you get better results.
